parachain/pallets/bridge/src/channel/inbound.rs

```diff
@@ -42,6 +43,8 @@ impl<T: Config> InboundChannel<T::AccountId> for BasicInboundChannel<T> {
 			Ok(())
 		})?;
 
+		T::RewardRelayer::pay_relayer(&T::RewardsAccount::get(), relayer, 10.into());
```

Comment:
this shouldn't be here for basic inbound?

Reply:
can we put the amount here into a config somewhere?

ideally something that can eventually be modified by governance too, though can defer that if its not quick and easy

Reply:
Yeah this will be all sorted out in the next PRs related to Governance & the incentivized bridge. Making fees configurable is a bit out-of-scope for this current PR, and I'm actually just gonna set the reward to `0` for now.